source/ember-inspector/data.md

```diff
@@ -1,40 +1,33 @@
-If you are using a data library that supports the Ember Inspector such
-as Ember Data, you can inspect your models by clicking on the `Data` menu.
+If you are using Ember Data, you can inspect your models by clicking on the `Data` menu.
```

Comment:
Given that we're considering the Guides to primarily document using Ember with Ember Data, I would even so eliminate the entire "If you are using Ember Data" bit and start with "You can inspect...".